_licenses/agpl-3.0.txt

```diff
@@ -4,7 +4,7 @@ nickname: GNU AGPLv3
 redirect_from: /licenses/agpl/
 source: http://www.gnu.org/licenses/agpl-3.0.txt
 
-description: The GNU GPL family of licenses is the most widely used free software license and has a strong copyleft requirement. When distributing derived works, the source code of the work must be made available under the same license. GNU AGPLv3 is distinguished from GNU GPLv3 in that hosted services using the code are considered distribution and trigger the copyleft requirements.
+description: Permissions of this strongest copyleft license are conditioned on making available complete source code of licensed works and modifications, which include larger works using a licensed work, under the same license when distributed. Copyright and license notices must be preserved. Contributors provide an express grant of patent rights to users. Additionally, when a modified version is used to provide a service over a network, the complete source code of the modified version must be made available to users of the service.
```

Comment:
re expectation of knowing what "copyleft" is; I agree! But it's always been on the [home page](http://web.archive.org/web/20130716093023/http://choosealicense.com/) which I think is far more important to be understandable. Suggestions welcome on how to get rid of it from either or both places.

On the bit like legalese bit, I agree. But it's hard to distinguish between licenses with some degree of accuracy without sounding like that. But I'd love to improve it. I think the main mitigation is that most users shouldn't get to /licenses, they should select one of the licenses on the home page, and among those the distinctions are more stark, so one can be accurate and very high level.

I've heard the phrase explain to me like I'm 5 before but never looked at a site dedicated to the concept. Not sure the reddit succeeds. I'd love to see proposed explanations of the main licenses as if reader were 5. Of the current descriptions, I'd say only MIT comes close, and that description has by far the easiest time of it (yet I don't know what "proper attribution" means). I think I'll open an issue with that challenge.

Finally I think there's some value in having an accurate and brief summary of each license even if it reads a bit like legalese, as those details matter and are really hard to extract through cultural osmosis or even just reading a license. But possibly that should be different from the description, especially if we had EILI5 descriptions.